.github/pr-commenter.yml

```
@@ -0,0 +1,67 @@
+comment:
+  on-update: recreate
+  header: |
+    Thank you for contributing to `exercism/elixir` :purple_heart: :tada:. This is an automated PR comment :robot: for the **maintainers** of this repository that helps with the PR review process. You can safely ignore it and wait for a maintainer to review your changes.
+
+    Based on the files changed in this PR, it would be good to pay attention to the following details when reviewing the PR:
+
+  footer: |
+    ---
+    Automated comment created by [PR Commenter](https://github.com/angelikatyborska/pr-commenter-action) :robot:.
+
+  snippets:
+    - id: any-file-changed
+      files:
+        - '*'
+        - '**/*'
+      body: |
+        - **General steps**
+            - :trophy: Does this PR need to receive a label with a modifier (`reputation/contributed_code/{minor,major}`)? (A medium reputation amount is awarded by default, see [docs](https://github.com/exercism/docs/blob/main/product/reputation.md#4-creating-pull-requests))
+
+    - id: any-exercise-changed
+      files:
+        - 'exercises/**/*'
+      body: |
+        - **Any exercise changed**
+            - :bust_in_silhouette: Does the author of the PR need to be added as an author or contributor in `<exercise>/.meta/config.json` (see [docs](https://github.com/exercism/docs/blob/main/building/tracks/practice-exercises.md#file-metaconfigjson))?
+            - :microscope: Do the [analyzer](https://github.com/exercism/elixir-analyzer/tree/main/test/elixir_analyzer/test_suite) and the [analyzer comments](https://github.com/exercism/website-copy/tree/main/analyzer-comments/elixir) exist for this exercise? Do they need to be changed?
+            - :scroll: Does the design file (`<exercise>/.meta/design.md`) need to be updated to document new implementation decisions?
+
+    - id: practice-exercise-changed
+      files:
+        - 'exercises/practice/**/*'
+      body: |
+        - **Practice exercise changed**
+            - :evergreen_tree: Do `prerequisites`, `practices`, and `difficulty` in `config.json` need to be updated?
+            - :teacher: Are the changes in accordance with the community [problem specifiations](https://github.com/exercism/problem-specifications/tree/main/exercises)?
```

```suggestion
            - :teacher: Are the changes in accordance with the community-wide [problem specifiations](https://github.com/exercism/problem-specifications/tree/main/exercises)?
```

As a maintainer, how do I know this? I feel like we need something to explain this more. It might be that a good doc is not yet for this, in which case we should add a placeholder doc and link to it.